### DISCUSSION TITLE
I don't know where this is supposed to go I'm new sorry

### DISCUSSION THREAD

**Discussion by 44cK3r** [skyline-emu/skyline#1224] [General]
Oppo Find X2 Pro Android 12 Official Update 2 (2nd Update) | Super Mario Odyssey | after starting the game from the in-game title screen vibration motors start going continuously no pattern or anything they just spin even after closing the emulator properly they don't stop until I ReBoot (Restart) My Phone

**Comment by 44cK3r** [comment]
Almost forgot on the same phone before android 12 came out for Australia on android 11 it wasn't doing the same thing the vibrations were in sync with the game and they stopped as soon as I closed the emulator sorry about that love you all so so so much

**Comment by yhak22** [comment]
Yes , that happened to me too ,
and it's expected cause the devs works to make the game work .
So just wait until they make it playable.

**Reply by 44cK3r** [reply]
All good thank you
